rest-api/instructions/routes/instructions-routes.js

```diff
@@ -0,0 +1,13 @@
+const router = require('express').Router();
+
+const {
+  updateInstructionByRecipeId,
+  deleteInstructionByRecipeId,
+  addInstructionByRecipeId
+} = require('../controllers/instructions-controllers');
+
+router.put('/:id', updateInstructionByRecipeId);
```

Comment:
Definitely need some validation here. And perhaps elsewhere. FE will be sending a token!

Reply:
Yep